client/src/components/Bookings/UpcomingBookings.tsx

```diff
@@ -0,0 +1,28 @@
+import Typography from '@material-ui/core/Typography';
+import Paper from '@mui/material/Paper';
+import BookingCard from './BookingCard';
+import moment from 'moment';
+import useStyles from './useStyles';
+import { useRequest } from '../../context/useBookingContext';
+
+const UpcomingBookings = (): JSX.Element => {
+  const classes = useStyles();
+  const { bookings } = useRequest();
+
+  return (
+    <Paper elevation={0} className={classes.upcomingBookings}>
+      <Typography className={classes.title} variant="h6" gutterBottom>
+        Upcoming Bookings:
+      </Typography>
+      {bookings.slice(1).map((booking) => (
```

Comment:
Would have loved to see this filtering done outside of the return statement (e.g. [`line 12` in `PastBookings.tsx`](https://github.com/hatchways/team-reindeer/blob/0e73a3b4f8e5b8ae9984a76802383df7f3b4ee54/client/src/components/Bookings/PastBookings.tsx#L12)). Also is the filtering logic complete? I would have expected a date comparison just like the one in the `PastBookings` component.

Reply:
Thanks, @ugur-akin for letting me know.